src/flow-manager.c

```diff
@@ -906,8 +906,8 @@ static TmEcode FlowManager(ThreadVars *th_v, void *thread_data)
             struct timeval cond_tv;
             gettimeofday(&cond_tv, NULL);
             struct timeval add_tv;
-            add_tv.tv_sec = 0;
-            add_tv.tv_usec = (sleep_per_wu * 1000);
+            add_tv.tv_sec = sleep_per_wu / 1000;
```

Comment:
Instead of saying we want 1 005 000 us, I say we want 1 second and 5000 microseconds
This avoids the possible overflow when doing sleep_per_wu * 1000